src/include/daos_types.h

```diff
@@ -352,15 +352,6 @@ typedef struct {
 
 typedef uint64_t	daos_epoch_t;
 
-static inline daos_epoch_t
-daos_ts2epoch(void)
-{
-	struct timespec ts;
-
-	clock_gettime(CLOCK_REALTIME, &ts);
-	return ts.tv_sec * 1e9 + ts.tv_nsec;
-}
-
```

Comment:
Do we still provide a function to convert epoch to struct timespec? That's definitely something that a end user might want (e.g. to extract snapshot time).